tests/comprehension_dict.py.uast

```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
```

Comment:
Updated. I added line numbers to some nodes that didn't have them in the native AST (which set them at 0 in the UAST). This fixes all of the previous errors except for the new `empty.*` test (expected, since it's empty, it's tests that the Python driver doesn't crash on this case like happened with a previous solved issue) and the `string_fstring.*`. 

This last one does the curious thing of setting the line numbers the code inside the fstring's {...} as starting from line 1. Shouldn't matter much because the next PR should checks and fix all those "questionable" locations of the native AST.